src/main/java/ua/lviv/iot/parkingServer/model/Vehicle.kt

```diff
@@ -0,0 +1,24 @@
+package ua.lviv.iot.parkingServer.model
+
+import ua.lviv.iot.parkingServer.model.enums.VehicleType
+
+data class Vehicle(
+    var vehicleId: Long,
+    var number: String,
+    var typeOfVehicle: VehicleType,
+    var durationOfUseOfParkingSpot: Double, // in hours
```

Comment:
Maybe you should use `Duration` instead of `Double`

https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html

Reply:
✅